major_tom/major_tom.py

```diff
@@ -240,3 +241,89 @@ def build_endpoints(self):
                 "files": files
             }
         })
+
+    def download_staged_file(self, gateway_download_path):
+        if self.https:
+            download_url = "https://"
+        else:
+            download_url = "http://"
+        download_url = download_url + self.host + gateway_download_path
+        r = requests.get(download_url,headers=self.headers)
```

Comment:
*nit*: Consider running this through a linter. There should be spaces after commas.